action.yml

```diff
@@ -29,50 +29,63 @@ inputs:
 runs:
   using: "composite"
   steps:
-  - name: Setup GITHUB_PATH for script
-    run: |
-      echo "${{ github.action_path }}" >> $GITHUB_PATH
-      echo "${{ inputs.working-directory }}"
-    shell: bash
-  - name: Grant permissions
-    run: chmod +x ${GITHUB_ACTION_PATH}/install.sh
-    shell: sh
-  - id: keploy-test-report
-    name: Run Script
-    run:  |
-      ${GITHUB_ACTION_PATH}/install.sh
-      ${GITHUB_ACTION_PATH}/install.sh > ${GITHUB_WORKSPACE}/${WORKDIR}/report.txt
-      cat ${GITHUB_WORKSPACE}/${WORKDIR}/report.txt
+    - name: Setup GITHUB_PATH for script
```

Comment:
extra space

Reply:
![image](https://github.com/keploy/testGPT/assets/114267538/abe6b6f0-2e5b-4fe2-a093-393a9221a64f)

According to this YAML checker, the indentation is correct and it is a valid yaml. Do you mind testing the workflow again?